src/mysql/create.sql

```diff
@@ -69,12 +69,28 @@ CREATE TABLE MentorExperience (
 -- Evidence of a user that is a mentor that proves them a previous intern.
 CREATE TABLE MentorEvidence (
   mentor_id INT NOT NULL,
+  approvals INT,
+  is_approved BOOLEAN,
+  is_rejected BOOLEAN,
```

Comment:
do we need is_rejected in evidence? wouldn't it be easier to just have the is_rejected in the Approval? what difference is there?

Reply:
Omar, just making sure you see this comment before I give the approval!

Reply:
Whoops, I did miss it, thanks for the bump! In another branch I do notice having is_approved and is_rejected in this table is unnecessary, so I change it to just is_reviewed.